client/src/components/location/Cities.js

```diff
@@ -24,14 +24,11 @@ class Cities extends Component {
   }
 
   render() {
-    let filteredCities = [];
-    const cities = this.props.cities || [];
-    const search = this.state;
+    let cities = this.props.cities || [];
```

Comment:
Just reducing the variable down to `cities` to make this a little cleaner.

Reply:
the need of this auxliar variable called filteredCities its because when trying to mutate cities which is the cities list comming from this.props.cities   I get an error saying    "cities" is read-only in the line: 
cities = cities.filter(city => {
In your next change the next line 30